src/test/java/com/solvd/movie/persistence/EsMovieRepositoryIT.java

```diff
@@ -0,0 +1,114 @@
+package com.solvd.movie.persistence;
+
+import com.solvd.movie.model.EsMovie;
+import com.solvd.movie.model.criteria.SearchCriteria;
+import com.solvd.movie.service.impl.MovieFactory;
+import integration.EsContainer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.util.List;
+
+@SpringBootTest
+@Testcontainers
+public class EsMovieRepositoryIT {
+
+    @Container
```

Comment:
@hizmailovich This logic along with `@DynamicPropertySource` should be moved to a separate base class, since some other integration test classes will use the same configuration

thus, it will help to avoid code duplication